src/main/java/ch/njol/skript/expressions/ExprPower.java

```diff
@@ -0,0 +1,137 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.FireworkExplodeEvent;
+import org.bukkit.event.entity.HorseJumpEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+
+@Name("Power")
+@Description("Gets the power/force within a supported event.")
+@Examples("power is greater than or equal to 2")
+@Since("2.4")
+public class ExprPower extends SimpleExpression<Number> {
+
+	static {
+		Skript.registerExpression(ExprPower.class, Number.class, ExpressionType.SIMPLE, "power");
+	}
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
```

Comment:
big oof
```java
	@Override
	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
		if (!ScriptLoader.isCurrentEvent(HorseJumpEvent.class, FireworkExplodeEvent.class)) {
			Skript.error("The power expression can only be used in a horse jump or firework explode event");
			return false;
		}
		return true;
	}
```

Reply:
That's more lines of code big oof

Reply:
If you haven't seen it yet, i changed the message. Why would you show a class name to users, used "or" instead of "and" and i made the message shorter.

The code format is like that in other elements, so not good to change in a single file

Reply:
Smaller code is always better, it's in literally every programming guide. Here is a link you can check out on your free time, it's Google's official Java guide, and it's one of the more trusted and reliable ones https://google.github.io/styleguide/javaguide.html

Reply:
Where it says give more importance to line number than readability? 
![image](https://user-images.githubusercontent.com/19474570/62195452-b24daa00-b384-11e9-8412-e8e134150094.png)

Again the format i sent is used in all others elements, should be considered as a standard and shouldn't be changed in a few files. It will just confuse the developers.

Reply:
Alright, well can you pin point what makes the code unclear for me please, so we can get over your rambling disease.

Reply:
I already did actually, but i found another good idea; the format you used won't be usable if there are multiple conditions, for example: https://github.com/SkriptLang/Skript/blob/ffac630718832b0bfcda216a351597362f40f5e6/src/main/java/ch/njol/skript/effects/EffCancelDrops.java#L65-L84

There can be many other different situations. So yeah, it will make it slower to read/write the code when you are used to use the original format.

Reply:
The ScriptLoader#isCurrentEvent supports multiple arguments, it's called a vararg, what you're trying to do there is redundant and useless as you're just trying to look smart. You can read up on them here at Oracles documentation https://docs.oracle.com/javase/6/docs/technotes/guides/language/varargs.html to educate yourself.

Reply:
lol you are funny